examples/AutoScrollExample_changed.js

```diff
@@ -0,0 +1,180 @@
+/**
+ * Copyright Schrodinger, LLC
+ */
+
+'use strict';
+
+import FakeObjectDataListStore from './helpers/FakeObjectDataListStore';
+import { ImageCell, LinkCell, TextCell } from './helpers/cells';
+import { Table, Column, DataCell, Plugins } from 'fixed-data-table-2';
+import React from 'react';
+import { FpsView } from 'react-fps';
+
+class AutoScrollExample extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      dataList: new FakeObjectDataListStore(10000),
+      scrollTop: 0,
+      scrollLeft: 0,
+      autoScrollEnabled: true,
+      horizontalScrollDelta: 0,
+      verticalScrollDelta: 0,
+      columns: [],
+      columnGroups: [],
+      columnsCount: 10000,
+    };
+    const cellRenderer = (props) => (
```

Comment:
This example should also work without any change, even when not using legacy table.

Reply:
Doesn't seems to be working correctly for me.
<img width="1505" alt="Screenshot 2023-06-29 at 12 51 32 AM" src="https://github.com/schrodinger/fixed-data-table-2/assets/89158056/131058c9-cae9-4763-904b-c29570f4e16d">

Note that cell-render could be anything i.e DataCell, TextCell, LinkCell, ImageCell or any customized cell or raw data, and this example should render same table as before for each case.

Reply:
Updated the auto scroll example